frb_codegen/assets/integration_template/shared/REPLACE_ME_RUST_CRATE_DIR/Cargo.toml.template

```diff
@@ -8,3 +8,5 @@ crate-type = ["cdylib", "staticlib"]
 
 [dependencies]
 flutter_rust_bridge = REPLACE_ME_RUST_FRB_DEPENDENCY
+log = "0.4.20"  # Optional - can be removed if you do not want logging
+chrono = "0.4.23"  # Optional - can be removed if you do not want logging (needed only for the default log output)
```

Comment:
nit: if you want to use chrono only in the macro, try `flutter_rust_bridge::for_generated::chrono` (and at the same time export it in frb_generated.rs)

Reply:
Ah, that would be great - bothered me to having to add this second dependency.

But I didn't understand how? 

> flutter_rust_bridge::for_generated::chrono

I can't find it - your didn't mean
```Rust
    fn _default_log_fn (record: Log2DartLogRecord) {
      let timestamp = flutter_rust_bridge::for_generated::chrono::Local::now();
```?
I only use it at [line 27 in the macro](https://github.com/patmuk/flutter_rust_bridge/blob/b561524bcae9466a68c72dc845c2c657fcd78ffc/frb_rust/src/misc/logging.rs#L27)

> at the same time export it in frb_generated.rs

Do you mean changing `frb_codegen` so it contains `pub use flutter_rust_bridge::for_generated::chrono`?

Reply:
1. add a line mimicking https://github.com/fzyzcjy/flutter_rust_bridge/blob/36e702332e1fb9855a28472c2e060a46d8063daf/frb_rust/src/for_generated/mod.rs#L70
2. write `flutter_rust_bridge::for_generated::chrono::Something`

Reply:
Ah, great, that worked :)
I needed to add `chrono` to the default dependencies in `frb_rust`.